src/analyzer/protocol/tcp/TCP_Reassembler.h

```diff
@@ -1,13 +1,18 @@
 #pragma once
 
+#include "analyzer/Analyzer.h"
 #include "Reassem.h"
 #include "TCP_Endpoint.h"
 #include "TCP_Flags.h"
 
 class BroFile;
 class Connection;
 
-namespace analyzer { namespace tcp {
+namespace analyzer {
+
+class Analyzer;
```

Comment:
Isn't this forward-declared in `analyzer/Analyzer.h` which you added?

Reply:
Right, first I added the include to make it build, then I decided I can replace it with a forward declaration, but then forgot to remove the include... fixed now.